src/utils/hmac-validator.ts

```diff
@@ -20,13 +19,10 @@ export function stringifyQuery(query: AuthQuery): string {
   return querystring.stringify(orderedObj);
 }
 
-export function generateLocalHmac({
-  code,
-  timestamp,
-  state,
-  shop,
-  host,
-}: AuthQuery): string {
+export function generateLocalHmac(
+  apiSecretKey: string,
```

Comment:
Putting a note here that this is changed in another branch, so these values can be obtained from config, when `generateLocalHmac` is attached using a `create__(config)` type function.